### PR TITLE
Improve Heroku deployment documentation

### PR DESCRIPTION
**Pull request by gabriel-dantas98**
Signed-off-by: Gabriel Dantas <gabriel.dantas98@outlook.com.br>

## Hey, I just made a Pull Request!

https://github.com/backstage/backstage/issues/9281

#### :heavy_check_mark: Checklist

- [ ] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [x] Added or updated documentation
- [ ] Tests for new functionality and regression tests for bug fixes
- [ ] Screenshots attached (for UI changes)
- [x] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

File: .changeset/silly-bees-grin.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
```
Comment (jhaals) [inline .changeset/silly-bees-grin.md:1]: This file is not needed for non package changes such documentation.

**Review by jhaals — COMMENTED** [review]
Thanks for approving the documentation 🥇 
Just delete the changeset file and this should be good to go!

File: .changeset/silly-bees-grin.md (hunk 1 above)
Comment (gabriel-dantas98) [reply .changeset/silly-bees-grin.md]: Tks 😉 I'II delete it

**Review by jhaals — COMMENTED** [review]
Thanks for removing the changeset. The signed-off-by line is needed for the last commit.
You can amend your commit with `--signoff` or see https://github.com/backstage/backstage/pull/9297/checks?check_run_id=5036355933 for more info.

**Comment by gabriel-dantas98** [comment]
I think I got a little confused in the commands I'll try to fix it 😞

**Comment by gabriel-dantas98** [comment]
I think it's better to open a new Pull Request :(

**Pull request closed without merging**